rfc/0011-connection-string.md

```diff
@@ -64,9 +67,12 @@ All key/value pairs should use lower-case, underscore seperated names.
   - http://fqdn
   - http://fqdn?key=value
   - couchbases://fqdn
+  - couchbase://[3ffe:2a00:100:7031::1]
```

Comment:
Should add an IPv6 example after line 64 and line 65 above, right?  These are the RFC-5952 examples.

Reply:
What additional examples would I be adding?

Reply:
There are no examples of how you'd do comma separated multiple hosts with IPv6, but there are examples with dotted quads and even dotted quads with port numbers.